src/views/NodeDetail.vue

```diff
@@ -82,23 +83,26 @@
                     class="pt-0"
                     color="transparent"
                   >
-                    <v-container fluid pa-0>
+                    <v-container
+                      fluid
+                      pa-0
+                    >
```

Comment:
```suggestion
                    <v-container
                      fluid
                      class="pa-0"
                    >
```

Reply:
Same as above.

Reply:
This seems to work, which makes the syntax a bit nicer for us, but I can't seem to find this documented anywhere. Maybe I'm just missing it in their docs, but all of their examples use the `class="..."` syntax. For that reason, I think it's better to stick to the recommended way of doing it, instead of using this non-standard way. I'm okay leaving this PR as is, just thought I'd throw my two cents in. I'd also be curious if you had a link to documentation that references this syntax @waxlamp.